target/linux/airoha/dts/an7581-gemtek-w1700k.dts

```diff
@@ -0,0 +1,241 @@
+/dts-v1/;
+
+/memreserve/ 0x80000000 0x200000; // ATF
+
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include "an7581.dtsi"
+
+/ {
+	model = "Gemtek W1700K";
+	compatible = "gemtek,w1700k", "airoha,an7581", "airoha,en7581";
+
+	aliases {
+		serial0 = &uart1;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200 earlycon";
+		stdout-path = "serial0:115200n8";
+		linux,usable-memory-range = <0x0 0x80200000 0x0 0x1fe00000>;
+	};
+
+	memory@80000000 {
+		device_type = "memory";
+		reg = < 0x0 0x80000000 0x0 0x80000000>;
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
```

Comment:
Please try to use `gpio-keys` instead. Then, `poll-interval` property can be removed.